decoder/vaapidecoder_h264.cpp

```diff
@@ -89,6 +89,18 @@ static VAProfile getH264VAProfile(H264PPS * pps)
     case 100:
         profile = VAProfileH264High;
         break;
+    case 110:
```

Comment:
Agree, since we also can't support SVC and MVC profiles right now. Thus means only profiles 66/77/100 supported.